packages/clerk-js/src/utils/retrieveCaptchaInfo.ts

```diff
@@ -0,0 +1,22 @@
+import type Clerk from '../core/clerk';
+import createFapiClient from '../core/fapiClient';
```

Comment:
☁️  i'm kind of skeptic about having a dependency of `core` inside `utils`. It may result in  a cyclic dependency in the future.